app/locales/en/translations.js

```diff
@@ -248,6 +248,11 @@ export default {
             support: `Support`,
             contact: `Contact`
         },
+        'permission-language':{
```

Comment:
I agree with that, but the translation file should still never contain duplicate sentences. The conditional logic should be in the template. There is a way to pass the name of the provider from the template to the translation.